### DISCUSSION TITLE
23.1.1 - Import/Export of settings but can't find it in the UI?

### DISCUSSION THREAD

**Discussion by Marmeladenbrot** [dbeaver/dbeaver#20410] [Q&A] [answered]
Hi, according to https://github.com/dbeaver/dbeaver/issues/17368 and the changelog ("Users can export DBeaver settings and import them") there now is a function to import and export the DBeaver settings.

But I can't find anything in the UI?

**Comment by Destrolaric** [comment, marked as answer]
Hi, you can find them in `File -> Import/Export -> Workspace`
![image](https://github.com/dbeaver/dbeaver/assets/20579256/a35f5248-78b8-4bc5-b20e-464e42494108)
